modules/nlu/src/backend/api.ts

```diff
@@ -74,7 +74,10 @@ export const registerRouter = async (bp: typeof sdk, app: NLUApplication) => {
   router.get('/health', async (req, res) => {
     // When the health is bad, we'll refresh the status in case it has changed (eg: user added languages)
     const health = await app.getHealth()
-    res.send(health)
+    if (!health) {
+      return res.status(404).send('NLU Server is unreachable')
+    }
+    return res.send(health)
```

Comment:
return not necessary since its the last statement. Shortcut: `health ? res.send(health) : res.status(404)`

Reply:
I personally prefer this way than with a ternary operator but it is indeed easier to read and less code

Reply:
seems more like a personal pref than a real code quality concern guys